components/chat/index.js

```diff
@@ -1,27 +1,34 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useRef } from 'react';
 import { MessageSquare, X } from 'react-feather';
 
-import colors from '../../styles/colors';
 import constants from '../../const';
-
-import { ButtonLink } from '../../uiComponents/button';
+import { ButtonLink } from '../../uiComponents';
+import './chatStyles.scss';
 
 const Chat = () => {
   const [isOpen, setIsOpen] = useState(false);
-  const [subject, setSubject] = useState();
-  const [body, setBody] = useState();
-
-  const getMailto = useCallback(() => {
-    return encodeURI(
-      `mailto:contact@api.gouv.fr?subject=[Question api.gouv.fr] ${subject}&body=${body}`
+  const subject = useRef(null);
+  const body = useRef(null);
+
+  const openMail = e => {
+    if (!subject || !subject.current || !body || !body.current) {
+      return;
+    }
+    const subjectVal = subject.current.value;
+    const bodyVal = body.current.value;
+    if (!subject || !body) {
+      return;
+    }
+    window.location.href = encodeURI(
+      `${constants.links.mailto.CHAT_QUESTION} ${subjectVal}&body=${bodyVal}`
```

Comment:
it's a bit weird to complete the subject value then set another variable. I'd rather write:

```
`${constants.links.mailto.SUPPORT}?subject=[Question api.gouv.fr] ${subjectVal}&body=${bodyVal}`
```